### PR TITLE
DevEx: Fix IRS superuser script and add documentation for how to use it

### PR DESCRIPTION
**Pull request by jimlerza**
The `irs-super-user` script stopped working, so I've converted it to use AWS SDK v3 and updated it to utilize our standard environment variables.

I've also added documentation for using the script to generate API tokens that we can use to test API v2 endpoints.

### CODE REVIEW

**Review by mmarcotte — APPROVED** [review]
love it!

**Review by jasonfu9966 — APPROVED** [review]
I have tested this and it works

File: scripts/irs-super-user.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
+// see ../docs/additional-resources/irs-super-user.md for detailed instructions
+
+const { requireEnvVars } = require('../shared/admin-tools/util');
```
Comment (zachrog) [inline scripts/irs-super-user.ts]: Nice switch over to Typescript! Can you make all of the imports/exports es6?

**Review by zachrog — CHANGES_REQUESTED** [review]
Looks good to me just need the imports and exports to be es6

**Review by zachrog — APPROVED** [review]

**Pull request merged**